src/UGridApi/UGrid.cpp

```diff
@@ -296,6 +310,56 @@ namespace ugridapi
         return exit_code;
     }
 
+    UGRID_API int ug_topology_define_double_variable_on_location(int file_id, int topology_id, int topology_type, int mesh_location, char const* variable_name)
+    {
+        int exit_code = Success;
+        try
+        {
+            if (ugrid_states.count(file_id) == 0)
+            {
+                throw std::invalid_argument("UGrid: The selected file_id does not exist.");
+            }
+
+            const auto name = ugrid::char_array_to_string(variable_name, ugrid::name_long_length);
```

Comment:
This variable `name` does not seem to be used. Can it be removed, or should line 326 use `name` instead of `variable_name`, or somehing else?